ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -361,22 +361,23 @@
 			};
 			name = "Debug-Development";
 		};
-		82B3057D290109C5000CC069 /* Debug-Development */ = {
+		82B3057F290109E5000CC069 /* Debug-Development */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9740EEB21CF90195004384FC /* Debug.xcconfig */;
 			buildSettings = {
-				APP_DISPLAY_NAME = "[DEV] Flutter Template Riverpod";
+				APP_DISPLAY_NAME = "DevFlutter Template Riverpod";
```

Comment:
Did you change this name? I think [DEV] Flutter Template Riverpod is better than DevFlutter

Reply:
yes i changed with [] now for both dev and prod